Mention JSBI as a new peer dependency in the 1.0 release notes

With the [v1.0.0-rc.1](https://github.com/awslabs/amazon-qldb-driver-nodejs/releases/tag/v1.0.0-rc.1) release, the ion-js dependency was upgraded to 4.0.0. The 4.0.0 version has a breaking change which involved making a dependency into a peer dependency. PRs for reference:

- https://github.com/amzn/ion-hash-js/issues/47
- https://github.com/amzn/ion-js/pull/589

If you miss the peer dependency warning when installing the RC and don't install jsbi, you may get the 17 build errors I got that looked a lot like this:

```sh
node_modules/ion-js/dist/commonjs/es6/IonWriter.d.ts:19:18 - error TS2307: Cannot find module 'jsbi'.

19 import JSBI from "jsbi";
```

JSBI is not a module I'm familiar with, but judging by how new it is [here](https://www.npmjs.com/package/jsbi) there's probably a good chance I'm not alone. **I'm opening up this issue to ask that the additional step of installing jsbi gets mentioned in the release notes and/or the README.**

Thank you for the great work on the 1.0 release! I thoroughly enjoyed deleting blocks of Reader step logic today. 🎉

--- a/README.md
+++ b/README.md
@@ -35,12 +35,14 @@ To use the driver, in your package that wishes to use the driver, run the follow
 
 ```npm install amazon-qldb-driver-nodejs```
 
-The driver also has aws-sdk and ion-js as peer dependencies. Thus, they must also be dependencies of the package that
+The driver also has aws-sdk, ion-js and jsbi as peer dependencies. Thus, they must also be dependencies of the package that
 will be using the driver as a dependency.
 
 ```npm install aws-sdk```
 
 ```npm install ion-js```
+
+```npn install jsbi```
 
 Then from within your package, you can now use the driver by importing it. This example shows usage in TypeScript 
 specifying the QLDB ledger name and a specific region:

--- a/README.md
+++ b/README.md
@@ -42,7 +42,7 @@ will be using the driver as a dependency.
 
 ```npm install ion-js```
 
-```npn install jsbi```
+```npm install jsbi```
 
 Then from within your package, you can now use the driver by importing it. This example shows usage in TypeScript 
 specifying the QLDB ledger name and a specific region:

